command/whitelist/show/show.go

```diff
@@ -0,0 +1,41 @@
+package show
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	showCmd := &cobra.Command{
+		Use:     "show",
+		Short:   "Command for listing all whitelists",
```

Comment:
This description needs to be changed to something like `Displays whitelist information`

Reply:
Fixed c92eb264615dcc30dfac51fd610892100fac7f86